site/releases/2024-dec-19/release-notes.qmd

```diff
@@ -0,0 +1,411 @@
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+We've extended the available options for custom analytics further — you can now sort bar chart visualizations and retrieve a count percentage metric for all types of visualizations.
+
+:::
+
+::: {.w-30-ns .tc}
+[Manage custom reports](/guide/reporting/manage-custom-reports.qmd){.button}
+
+:::
+
+::::
+
+<!---
+PR #1076: [7333] - chore: add UI changes for revision history
+URL: https://github.com/validmind/frontend/pull/1076
+Labels: enhancement
+--->
+#### Improved revision history within content blocks
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+We've improved the way revision history displays within content blocks, allowing you to more easily review changes to model documentation, validation reports, and ongoing monitoring reports. 
+
+:::
+
+::: {.w-30-ns .tc}
+[Collaborate with others](/guide/model-documentation/collaborate-with-others.qmd#revision-history){.button}
+
+:::
+
+::::
+
+![Example revision history for a content block](revision-history.png){width=80% fig-alt="A screenshot of example revision history for a content block" .screenshot}
+
+<!---
+PR #1092: [SC-7471] adding custom field permission check
+URL: https://github.com/validmind/frontend/pull/1092
+Labels: enhancement
+--->
+#### Granular role permissions for inventory fields
+
+::: {.column-margin}
+![Setup for an inventory field only editable by developers](developer-only-field.png){fig-alt="A screenshot of the setup for an inventory field only editable by developers" .screenshot}
+:::
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-70-ns}
+You can now gate edit access to specific model inventory fields to certain roles via the **[write permissions]{.smallcaps}** setting.
+
+:::
+
+::: {.w-30-ns .tc}
+[Manage model inventory fields](/guide/model-inventory/manage-model-inventory-fields.qmd){.button}
+
+:::
+
+::::
+
+- Leave this field blank to leave permissions as default.
+- By default, model owners and validators have write access. Model developers have read-only access.[^4]
+
+## Bug fixes
+
+### {{< var validmind.platform >}} (v1.27.17)
+<!---
+PR #1106: Deleting a finding should remove it from a validation report
+URL: https://github.com/validmind/frontend/pull/1106
+Labels: bug
+--->
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+- Previously, deleting a finding did not remove it from linked validation reports. 
+- Now, deleting a finding will show a confirmation warning and subsequently remove it from any linked reports.
+
+::: {.tc}
+[Add and manage model findings](/guide/model-validation/add-manage-model-findings.qmd#delete-model-findings){.button}
+
+:::
+
+:::
+
+::: {.w-40-ns .tc}
+![Deletion confirmation dialogue for a finding linked to a validation report](delete-linked-finding.png){fig-alt="A screenshot of the deletion confirmation dialogue for a finding linked to a validation report" .screenshot}
+
+:::
+
+::::
+
+## Documentation (v1.27.17)
```

Comment:
```suggestion
## Documentation
```

The docs are technically versioned against the ValidMind Library but we shouldn't include that version here as it has no real meaning to our users, especially since this version of the documentation also covers frontend features and the versioning scheme is about to change (switching to Spencer's 'meta version' that includes other versions).

Reply:
Gotcha, I was just doing it for uniformity but I can remove this no problem.